### DISCUSSION TITLE
数据库访问问题Internal Server Error

### DISCUSSION THREAD

**Discussion by qingfengJava** [hhyo/Archery#2247] [General]
项目正常部署，通过放行域名后可以正常登录系统，整个系统通过域名进行访问，服务之间都是使用内网进行访问，mysql和Redis都是使用的云服务器，有服务名，现在的问题是数据库实例可以创建，但是操作数据的时候比如查询，SQL上线等会报Internal Server Error问题。这个我猜想是goinception在访问过程中受限制了，但是我使用域名和服务器名这种方式，可以怎么进行放行，这种问题具体可以怎么解决。
![image](https://github.com/hhyo/Archery/assets/97382270/7e95d9da-cc1d-4199-bb0e-be32e75ea3e4)

![image](https://github.com/hhyo/Archery/assets/97382270/b213469f-443b-468b-aa70-b74a4643c68e)

**Comment by hhyo** [comment]
查看下日志文件，500会有异常记录

**Reply by qingfengJava** [reply]
![image](https://github.com/hhyo/Archery/assets/97382270/86b9a316-c61d-4515-a7fd-7a10b17f5696)

我查看了一下日志：有个django的问题，这里我不太懂，因为我们都是配的服务器的内网进行访问的，有没有可能是这个的问题，我在自己的服务器上用公网IP去配置就没有任何问题，这块麻烦帮忙解决一下。
![img_v2_c1cff833-6a6b-4ff4-8209-6e51e1f99a5g](https://github.com/hhyo/Archery/assets/97382270/70b60f30-8ff8-405a-9786-821f715824c0)
![image](https://github.com/hhyo/Archery/assets/97382270/33e9a871-0c45-4de1-a800-1ecfa15ab37e)

**Reply by hhyo** [reply]
错误已经很明显了，连不上redis

**Reply by qingfengJava** [reply]
这个是需要我配置Redis的白名单是吗

**Comment by qingfengJava** [comment]
> 错误已经很明显了，连不上redis

**Comment by qingfengJava** [comment]
感谢大佬，已经成功了
